jaeger_client/metrics/metrics.py

```diff
@@ -12,13 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import Any, Optional, Callable, Dict
+
+
 class MetricsFactory(object):
     """Generates new metrics."""
 
     def _noop(self, *args):
         pass
 
-    def create_counter(self, name, tags=None):
+    def create_counter(self, name: str, tags: Optional[Dict[str, Any]] = None) -> Callable:
```

Comment:
a. I think tags are str->str
b. I think it would be more useful to clearly define the Callable shape, it's kind of the major part of the API

Reply:
Can you please tell me the shape?

Reply:
My best guess it's (float)->None

Reply:
actually (int)->None for counter and float for Gauge

Reply:
done